images/miq-app/ose-assets/appliance-initialize.sh

```diff
@@ -1,15 +1,39 @@
 #!/bin/sh
 
+set -x
+
 [[ -s /etc/default/evm ]] && source /etc/default/evm
 
 # Source original OSE environment, generated by lifecycle hook script.
 
 [[ -s ${APP_ROOT}/tmp/ose-env ]] && source ${APP_ROOT}/tmp/ose-env
 
+if [[ -z $DATABASE_SERVICE_NAME ]]; then
```

Comment:
@fvillain These should never come empty if deploying from template, since they have default parameters injected.
As long as the container environment is passed correctly to systemd, we should not have to check further. Did I miss anything?